lib/commands/import/command.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "import",
+  "description": "Generates code for common use cases in Aurelia applications.",
+  "parameters": [
+    {
+      "name": "generator-name",
+      "description": "The name of the generator you want to run."
+    }
+  ]
+}
```

Comment:
We need to fix up the description and parameter info here so that it matches the command. This is what gets displayed when the user asks for help.